src/newSeed/baseStage.scss

```diff
@@ -528,6 +528,9 @@
               @include tokenIcon;
               height: 19px;
             }
+            &.projectLogo {
+              grid-template-columns: 0.1fr 1fr;
```

Comment:
The horizontal spacing in this element doesn't always look good at various page widths.  The icon isn't vertically centered.

(BTW I note there is another div here without its own class.)

I just committed changes that make the horizontal and vertical spacing/alignment more consistent and appealing.